src/Cli/dotnet/commands/RestoringCommand.cs

```diff
@@ -60,11 +60,11 @@ public class RestoringCommand : MSBuildForwardingApp
                 return null;
             }
 
-            IEnumerable<string> restoreArguments = new string[] { "-target:Restore" };
+            IEnumerable<string> restoreArguments = ["-target:Restore"];
             if (arguments != null)
             {
-                restoreArguments = restoreArguments.Concat(arguments.Where(
-                    a => !IsExcludedFromRestore(a) && !IsExcludedFromSeparateRestore(a)));
+                (var newArgumentsToAdd, var existingArgumentsToForward) = ProcessForwardedArgumentsForSeparateRestore(arguments);
+                restoreArguments = [.. restoreArguments, .. newArgumentsToAdd, .. existingArgumentsToForward];
```

Comment:
I haven't used the range operator much yet, but it looks like the documented usage doesn't have a space between the operator and the value. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#range-operator-

Not sure if there is an editor.config rule for this behavior yet.

Reply:
dotnet-format keeps reformatting it like that (and my editor applies the format rules on save) so it would just keep resetting itself if I changed this.